docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -241,20 +242,4 @@ mysql -u user -h host database -p
 ```
 and provide the password on prompt.
 
-To preview very large tables, use `\G` instead of `;` at the end of a query. This
- formats the query output nicely, so that you can read through it easier. 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+To preview very large tables, use `\G` instead of `;` at the end of a query. This formats the query output nicely, so that you can read through it easier. 
```

Comment:
this is a specific mysql commandline tip - 
```suggestion
MySQL commandline tip: try using `\G` instead of `;` at the end of a query - this will print one column value per line, which makes the query output easier to read. 
```